book/README.md

```diff
@@ -1,6 +1,6 @@
 # reth Book
 
-<img src="images/" style="border-radius: 20px">
+<img src="https://raw.githubusercontent.com/m0ham3dx/reth/main/assets/reth.jpg" style="border-radius: 20px">
```

Comment:
can we change this to a relative import `../assets/reth.jpg`?

Reply:
changes implemented -

Reply:
it seem 
```
<img src="../assets/reth.jpg" style="border-radius: 20px">
``` 
doesnt render for some reason when pushed to github pages. When I was reading the book on my local repo 

But it works when the image is imported as a full url 
```
https://raw.githubusercontent.com/paradigmxyz/reth/main/assets/reth.jpg
```
When the book gets published eventually, and the image is broken, I will submit another pr 

if it doesnt, then i will resolve this conversation. Until then no more prs on this issue from me

@mattsse

Reply:
okay, let's do the full url then

mind submitting another PR?

Reply:
PR #839